pkg/controller/csi_handler_test.go

```diff
@@ -624,7 +624,7 @@ func TestCSIHandler(t *testing.T) {
 			},
 			expectedCSICalls: []csiCall{
 				{"attach", "projects/UNSPECIFIED/zones/testZone/disks/testpd", testNodeID,
-					map[string]string{"partition": "0"}, noSecrets, readWrite, success, notDetached, noMetadata, 0},
+					map[string]string{"partition": ""}, noSecrets, readWrite, success, notDetached, noMetadata, 0},
```

Comment:
This is due to https://github.com/kubernetes/kubernetes/pull/74932

Reply:
It might be better to separate out this change into a separate commit in this PR (with the main commit focussing on the vendoring update).

Reply:
@msau42 so this is expected then

Reply:
Yes this is expected